docs/eventing/README.md

```diff
@@ -97,4 +97,4 @@ resources:
 
 ## Observability
 
-- [Eventing Metrics API](./metrics.md)
+- [Eventing Metrics API](../admin/collecting-metrics/eventing-metrics/metrics.md)
```

Comment:
do we want some reqs in the style guide about links? e.g. whether to use absolute or relative, and whether or not we need .md on the end?